src/java.base/share/classes/java/math/MutableBigInteger.java

```diff
@@ -1882,6 +1906,204 @@ MutableBigInteger sqrt() {
         }
     }
 
+    /**
+     * Calculate the integer square root {@code floor(sqrt(this))} and the remainder
+     * if needed, where {@code sqrt(.)} denotes the mathematical square root.
+     * The contents of {@code this} are <b>not</b> changed.
+     * The value of {@code this} is assumed to be non-negative.
+     * @throws ArithmeticException if the value returned by {@code bitLength()}
+     * overflows the range of {@code int}.
+     * @return the integer square root of {@code this} and the remainder if needed
```

Comment:
To emphasize, make use of `<em>` tags, not `<b>`.

By convention, a blank line precedes `@return`, which precedes `@throws` (see some public API examples).